.travis.yml

```diff
@@ -1,5 +1,13 @@
 language: rust
 sudo: required
+after_success: |
+  [ $TRAVIS_BRANCH = master ] &&
+  [ $TRAVIS_PULL_REQUEST = false ] &&
+  cargo doc &&
+  echo "<meta http-equiv=refresh content=0;url=neon/index.html>" > target/doc/index.html &&
```

Comment:
Here we redirect to `neon/index.html` since we're also going to be hosting the `neon-sys` and `cslice` crate docs.